File: 3-Leccion-Condicionales-Ciclos/E.Practica-Ciclos-ASINUS.java
import java.util.Scanner;

public class Main {
  public static void main(String[] args) {
    Scanner scanner = new Scanner(System.in);

    int N = scanner.nextInt();

    int suma = 0;
    for (int i = 0; i < N; i++) {
      int x = scanner.nextInt();
      suma += x;
    }

    System.out.println(suma);
  }
}

File: 3-Leccion-Condicionales-Ciclos/E.Practica-Ciclos-ASINUS.cpp
#include <iostream>
using namespace std;

int main() {
  int N;
  cin >> N;

  int suma = 0;

  for (int i = 0; i < N; i++) {
    int x;
    cin >> x;
    suma += x;
  }

  cout << suma << endl;
}